src/main/java/net/mcreator/element/types/GUI.java

```diff
@@ -50,6 +50,12 @@
 	public Procedure onTick;
 	public Procedure onClosed;
 
+	public int sx = 18;

```

Comment:
Overlay has grid settings too, so please make GridSettings inside element/parts and move these fields there and use GridSetting files in both mod elements and add support for overlays too.

Reply:
What class would GridSetting extends?

Reply:
No extends, just additional data class